pom.xml

```diff
@@ -188,6 +201,27 @@
 				<openMRSMinorVersion>1.10</openMRSMinorVersion>
 			</properties>
 		</profile>
+
+		<profile>
+			<id>Java 17</id>
+			<activation>
+				<jdk>17</jdk>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<version>3.2.2</version>
+						<configuration>
+							<argLine>--add-opens java.base/java.lang=ALL-UNNAMED
```

Comment:
Can we look into avoiding having to add this?

Reply:
We need these for some of the testing libraries. I think it would be hard to support both Java 8 and 17 if we want to avoid using these.